src/genie/libs/parser/utils/unittests.py

```diff
@@ -54,7 +54,7 @@ def get_operating_systems(_os):
     # Update and fix as more OS's converted to folder based tests
     if _os:
         return [_os]
-    return ["asa", "ios", "iosxe", "junos", "iosxr", "nxos"]
+    return ["asa", "ios", "iosxe", "junos", "iosxr", "nxos", "gaia"]
```

Comment:
It seemed `gaia` was removed when I fixed conflict on below PR. so added.
https://github.com/CiscoTestAutomation/genieparser/pull/408